index.bs

```diff
@@ -12086,8 +12086,7 @@ and [[#legacy-platform-object-set]].
             1.  [=Invoke the named property setter=] with |P| and |V|.
             1.  Return <emu-val>true</emu-val>.
     1.  Let |ownDesc| be <a abstract-op>LegacyPlatformObjectGetOwnProperty</a>(|O|, |P|, <emu-val>true</emu-val>).
-    1.  Perform [=?=] [=OrdinarySet=](|O|, |P|, |V|, |Receiver|) from step 3 onwards
-        with its internal "ownDesc" variable set to |ownDesc|.
+    1.  Perform [=?=] [=OrdinarySetWithOwnDescriptor=](|O|, |P|, |V|, |Receiver|, |ownDesc|).
```

Comment:
`|V|` should be `<emu-val>true</emu-val>` I suspect? And it also looks like we don't have a `|Receiver|`?